source/Interfaces/NetWorth/NetworkInterface.hpp

```diff
@@ -0,0 +1,93 @@
+/**
+ * This file is part of the Fall 2023, CSE 491 course project.
+ * @brief A networking interface that allows information to be sent across a  network
+ * @note Status: PROTOTYPE
+ **/
+
+#pragma once
+
+#include <cassert>
+#include <string>
+#include <vector>
+#include <SFML/Network/UdpSocket.hpp>
+#include <SFML/Network/Packet.hpp>
+
+#include "../../core/InterfaceBase.hpp"
+
+using namespace sf;
+
+namespace cse491 {
+    namespace netWorth{
+    /**
+     * TODO: Delete or incorporate
```

Comment:
It looks like this is used in the Client and Server interfaces, so I assume this is an old comment?

Reply:
Currently this class is not being used, as it is meant to be a parent class for both the client and server interface, however the class itself is not being used as of this PR. It will be incorporated in upcoming versions of the module but we wanted to leave a TODO there so we did not forget that.